### ISSUE TITLE
How to center some of the icons?

### ISSUE THREAD

**Issue by ElecM8** [jadeonking/Witcher-3-Theme-for-Discord#3]
Hey! First of all, a big thank you for this theme (using the Witcher 3 one) which is absolutely amazing.
I just wanted to ask a little help because I absolutely suck at css (In fact I never really wrote anything) and I wanted to modify that theme just a little if you don't disagree.

If you're okay with helping, here's my problem: https://i.imgur.com/rkSd2UY.png
I was wondering what should be modified if I wanted to center all of this. I don't find it very appealing to stick everything to the left, feels too "uneven" for me.
In all honesty, I've tried editing the "margin-left" on some things but then the notifications (the red bar on the right) would disappear, and I don't know what controls that...

Basically I'd like to center:
- The "friends" icon at the top
- The "online" text
- The servers' icons bellow, while keeping the messages notifications

Also, one last thing, I'd like to know how to add spaces between the servers icons, that's my personal taste but I find it weird to stick them all to each other.
Thank you very much! And thanks again for the theme, absolutely lovely :+1:

**Comment by jadeonking** [comment]
Hey so this line .guilds-wrapper .guilds .guild .guild-inner removing margin-left from it will move most server icons and the friends back to the center removing padding from this line .guilds-wrapper .guilds will also help as for the spaces between avatars I am still looking for it its been a long time since i edited this

**Comment by jadeonking** [comment]
is what your wanting right? this is duskcord btw witcher 3 will obviously be transparent
![ss](http://i.imgur.com/ub2slqn.png)

**Comment by jadeonking** [comment]
.guilds-wrapper .guilds .guild { 
.guilds-wrapper .guilds .guild+.guild { remove height from both

.guilds-wrapper .guilds .guild .guild-inner { remove margin left, padding left and padding top as well as top

.guilds-wrapper .guilds .guild.unread:before { remove everything except Background

.guilds-wrapper .guilds-add { remove margins left and top

.guilds-wrapper .guilds-error { I believe you remove right, margin top, and height

that should center all of it

everything will still be square is that wat u want?

**Comment by ElecM8** [comment]
Hey, thanks for your answer.
This is indeed what I want. I removed what you told me to remove, but it didn't achieve what I need (The icons are still stuck on the left of the screen, also there are no space between the last server icon and the "create server" placeholder

But since you pointed out what parts of the css need to be edited, I'll have a look and try to manage by myself. I'll update you when I'm done testing. Thanks for your help!

**Comment by ElecM8** [comment]
Hey @jadeonking 
Turns out I had deleted the wrong things at first, fixed it now. The icons are centered but I have two more issues now

- The position of the number that shows how many times you were mentioned/PM'd is on the middle of the icons: https://i.imgur.com/GNH8GyC.png
What part of the CSS do I have to look at to edit it?
- The number of online users is still on the left, what should I edit to change that?

Just point me to the .guilds-wrapper (or whatever else) that handles both of those, I'll figure it out. Thanks!

**Comment by jadeonking** [comment]
.guilds-wrapper .guilds .guild .badge { for the badge

.guilds-wrapper .guilds .friends-online { for the # online

**Comment by jadeonking** [comment]
also in the future i'm not sure if you were aware of this but if you do ctrl+shift+i you can easily find everything by using the inspector I also recommend using Notepad++ or Atom either is fine it will make editing CSS far easier
http://i.imgur.com/5ix3SLs.gifv

**Comment by ElecM8** [comment]
No I wasn't aware of that.
But thank you, this is a great tip, I'll probably use it in the future.
Anyway, I'm done editing your theme thanks to your help. Hopefully this "issue" will help someone else in the future.

Thanks again mate :+1:

**Comment by jadeonking** [comment]
no problem and enjoy
